setup.py

```diff
@@ -1,9 +1,5 @@
 import sys
 
-vi = sys.version_info
-if vi < (3, 8):
-    raise RuntimeError('uvloop requires Python 3.8 or greater')
-
```

Comment:
> "requires-python = '>=3.8.0'" in pyproject.toml is enough.

I'd keep this extra check. For people who uses Python < 3.8, they may also have a `pip` around year 2016 or earlier, which doesn't honor `requires-python`.